Example/SmileID/DocumentVerificationIdTypeSelector.swift

```diff
@@ -63,5 +63,10 @@ struct DocumentVerificationIdTypeSelector: View {
                 }
             }
         }
+        .onAppear {
```

Comment:
What's the difference between loading the services in the ViewModel's `init` vs `onAppear`?

Reply:
That makes sense. I didn't know about the `.task {  }` modifier, that sounds useful. And after thinking about it some more, moving the loading out of the init would make testing easier too